lib/legacy_activity.rb

```diff
@@ -0,0 +1,42 @@
+require "csv"
```

Comment:
As a class that was representing data not sourced by this service I felt that a home within lib/ would be more appropriate that models/. A models folder we use regularly and if it included both `activity` and `legacy_activity` that would create a lot of unnecessary confusion.